### DISCUSSION TITLE
First Star

### DISCUSSION THREAD

**Discussion by drakon660** [tye-v2/tye2#42] [Announcements]
![image](https://github.com/tye-v2/tye2/assets/3002778/4f46ec21-ebc0-46e5-95c6-fca8e107234f)

:)

**Comment by Mrxx99** [comment]
That's me :)
Found this because I looked at the old tye repo and found a link to this on a old open PR

**Reply by drakon660** [reply]
Be our gest, lots things to improve.

**Reply by razvangoga** [reply]
welcome! i think github linked back when i mentioned the PRs i merged - i was going to open an issue there to let people know about the fork, but Fowler beat me to it and archived the repo :))
